### ISSUE TITLE
It doesn't work on Ryzen 3000 cpu

### ISSUE THREAD

**Issue by gundamftw** [Tranquilite0/FF12RNGHelperPC#3]
I'm trying to get the Seitengrat bow from the sky ferry. I have a Ryzen 3900x and when I  have my character waited at the doorway, the "advance to spawn" counter will always jumps between 0-3, there was never a countdown.

**Comment by Tranquilite0** [comment]
I seriously doubt that your CPU is the problem here.

RNG values are consumed by NPC interactions in a very inconsistent and unpredictable manner such that it is always a bit of a gamble when trying to go through the door at the RNG value you need because all the NPCs moving around while you go through the door, and can consume RNG values at the last moment. My advice is to simply keep on trying until you get it right. With that said, there are a couple things you can try to make it more consistent (but still not guaranteed).

The first is to reduce the refresh interval of the RNG Helper app so that it updates more often and gives you a more accurate view of the RNGs current positon. Since you have a big beefy CPU its probably fine to change the refresh interval to something quick like 10ms.

The second is to try changing the game's frame-rate to 30FPS. I cant recall for sure if the NPC interactions are tied to the frame-rate, but in theory going from 60 to 30 fps will give the game half as many opportunities to mess things up while you go through the door transition.

Good luck!

**Comment by gundamftw** [comment]
I uploaded a video here, there is no countdown, the "advance to spawn" counter stays at 0 most of the time.

https://youtu.be/KSkLaa7pmIw

**Comment by Tranquilite0** [comment]
Ah yes, I see. It looks like you don't have the Seitengrat chest's properties configured into program. In order to predict when a chest will spawn, you need to enter in its spawn chance, and spawn position. In order to predict the right result from the chest after it has spawned, you need to enter in its gil chance and item chance, and (if you care about gil) maximum gil amount. The reason why your advance to spawn counter only seems to move between 0 and 3 is because you have the program set to look for a chest that spawns 50% of the time.

In other words, you need to enter in the following value for Chest 1 before the RNG helper can help you get your Seitengrat:
Spawn chance: 1
Spawn position: 1
Gil Chance: 80
Item 1 Chance: 95
Max Gil Amount: 10
And then make sure the Desired Item radio box has Item 2 Selected.

If you are still having trouble then I suggest following a guide like this one [https://www.youtube.com/watch?v=5Fb1v9lE5p8](https://www.youtube.com/watch?v=5Fb1v9lE5p8)

**Comment by gundamftw** [comment]
Thanks, I entered the correct number and got the bow in a few tries.

I have a suggestion, can you add all the rare items with their corresponding values in a drop down list  so that people don't need to look elsewhere for the chart?
